src/utils/pathnames.js

```diff
@@ -0,0 +1,5 @@
+export const pathnames = {
+  loginUrl: '/login-hayoung',
+  listUrl: '/list-hayoung',
+  detailUrl: '/detail-hayoung',
+};
```

Comment:
```suggestion
export const PATH = {
  login: '/login-hayoung',
  list: '/list-hayoung',
  detail: '/detail-hayoung',
};
```
고정 상수는 NAME_LIKE_THIS 컨벤션으로 해주세요 :)

Reply:
감사합니다! 수정했습니다!

https://github.com/wecode-bootcamp-korea/1-React-Webucks-1/pull/1/commits/dee019fbc105bd79e00f973b64232ea0d06f8467